pyproject.toml

```diff
@@ -47,15 +48,21 @@ classifiers=[
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.13"
+python = "^3.8"
```

Comment:
This was the requested change that motivated me to start poking around with this.